inference_rules.adoc

```diff
@@ -428,6 +428,81 @@ division allows calibration for quantization and does not allow any retraining.
 
+=== Network Division
+
+The Network division inherits all requirements from the Closed division and imposes further constraints.
+In the Network division the SUT is connected to the Loadgen system over a network fabric. The Query Dispatch Library (QDL) component is a submitter-implemented SUT proxy that runs on the Loadgen system. The Network division supports only the Datacenter suite.
+
+==== QDL Constraints
+
+* The QDL is not allowed to do any pre-processing.
+* If an SUT compresses its output, the QDL must decompress the output. Decompression is a timed operation. No other post-processing in the QDL is allowed.
+* The QDL is not allowed to batch queries.
+* The Name method's return value must contain the substring "Network SUT".
+* The Name method's implementation must include at least one round trip over the network. The Name method must not return until the round trip is complete.
+
+==== General Constraints
+
+MLPerf distinguishes between fabric interconnects and bus interconnects. Fabric interconnects are required and bus interconnects are forbidden.
+
+A fabric interconnect must:
+
+1. Work as out-of-the-box chassis-to-chassis interconnect
+1. Use wireless, copper, or fiber-optics media
+1. Suitable for connecting systems above 10 meters distance
+1. Use switch topology
+1. Be highly scalable, reliable, and fault-tolerant
+
+Currently permitted fabric interconnects are Ethernet, IEEE802.11, Infiniband, and 3GPP.
+
+Examples of forbidden bus interconnect include: PCIe/CXL/CCIX, Hypertransport, NVLink, QPI, UPI, and ICI (interchip interconnect).
+
+Additionally, any interconnect not listed in the permitted list is forbidden unless clearance is first obtained from the MLPerf Inference WG.
+
+The SUT and QDL must run on two physically separated and distinct systems.
+
+Fabric and protocol must be reported in the submission metadata. Submission metadata must be sufficient to determine OSI layers one through four of the submission's network stack.
+
+==== SUT over the Network and Setup Constraints
+
+* SUT parameters and configuration must be uniquely and specifically named in the submission results. 
+* Network interface characteristics must be added to the report, both at the SUT side and at the Loadgen system side, including networking components, and name any other components through which data passes between Loadgen and the SUT for reproducibility.
+
+==== Benchmarks and QSL Preprocessing
+
+Data formats for inputs and outputs are allowed to be compressed for network transmission, providing a tradeoff between compute and network bandwidth. Data transferred between the Loadgen system and the SUT can be compressed using one of the options from the following table for each benchmark. Compression is performed by QSL, and is untimed. The compression scheme needs approval by the Working Group, allowing compression schemes that will be suitable for production, so for example, very asymmetric schemes are not expected to be approved.
```

Comment:
@DilipSequeira Are these compression formats same as the ones already permitted for QSL? If so, maybe this section can be moved out to a place higher in the doc hierarchy?